### PR TITLE
Minor intro page improvements

### PR DESCRIPTION
**Pull request by Shiaulis**
Dropped all changes I felt make sense for your awesome home page. Feel free to approve only particular ones, I just decided to collect all of my ideas so far 😄 
You can see how it looks [here](https://github.com/Shiaulis/khaletska/tree/update)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-## [PORTFOLIO](https://khaletska.github.io/portfolio/)
+### You can find my full portfolio [here](https://khaletska.github.io/portfolio/)
```
Comment (Shiaulis) [inline README.md]: I didn't notice your porftolio link until I noticed it is an actual hyperlink. If you feel it is prominent and visible then let's no make this change

File: README.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
-### 1. About me:
-- Full-stack development student at [kood/Johvi](https://kood.tech/).
+### About me:
```
Comment (Shiaulis) [inline README.md:3]: does it make sense not to have numbers?

File: README.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+- Full-stack development student at [kood/Jõhvi](https://kood.tech/).
```
Comment (Shiaulis) [inline README.md:4]: I just love **Õ** letter 😄

File: README.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
 - iOS Intern at [Pipedrive](https://www.pipedrive.com/).
 - Passionate about turning stunning designs into seamless user experiences.
 - Technical mind with over 5 years of experience in various engineering roles.
 - Equally able to work independently and as a collaborative team player. 
 
-### 2. Favorite tools:
-- Mobile: [Swift](https://www.swift.org/)
+### Favorite tools:
+- Mobile: [Swift](https://www.swift.org/), [Objective-C](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Introduction/Introduction.html)
```
Comment (Shiaulis) [inline README.md:11]: the important technology was missed 🤨

File: README.md [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
 - Project management: [Jira](https://jira.atlassian.com/)
 - Always ready to dive into new technologies to ensure the best outcomes for all
 
-### 3. Education:
-- Full-stack development student at kood/Johvi
+### Education:
+- Full-stack development student at kood/Jõhvi
 - Master's degree in Space Research
-- Bachelor's degree in Mechanical Egineering
+- Bachelor's degree in Mechanical Engineering
```
Comment (Shiaulis) [inline README.md:22]: this is the only change I was going to propose 😥 and then didn't manage to stop 🫠

**Review by khaletska — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (khaletska) [reply README.md]: Let's leave it as it was

**Pull request merged**